src/examples/render.ts

```diff
@@ -0,0 +1,56 @@
+import { Renderer } from '../renderer/Renderer';
```

Comment:
General comment for this file. Is it usual to have code just at the top level of the file? (Like generating indices, normals and vertices)

Reply:
It's probably better to have it elsewhere, but I figured we can rearrange this once we have a better way of generating geometry (since any organization I do now is probably subject to change.)

Reply:
Although I think actually this is probably how end users will end up using it, at first anyway - probably they'll just start with a file and then maybe only split up stuff once it grows